test/integration/examples-bad/ode/adams/stanc.expected

```diff
@@ -1,6 +1,6 @@
   $ ../../../../../../install/default/bin/stanc bad_adams_control_function_return.stan
 
-Semantic error at file bad_adams_control_function_return.stan, line 13, columns 6-153:
+Semantic error at file bad_adams_control_function_return.stan, lines 13-15, column 6:
```

Comment:
I think this is the same comment as before, but multiple lines with the same column is just confusing here.

Reply:
So, print both columns?